website/src/components/wordpress.tsx

```diff
@@ -78,6 +81,12 @@ const parseOptions: HTMLReactParserOptions = {
           props["href"] = props["href"].slice(wordpressUrl.length)
         }
         return <Link {...props}>{domToReact(node.children, parseOptions)}</Link>
+      } else if (node.attribs["class"]?.includes("print-lesson")) {
+        return (
+          <div className={wpCss.wp}>
```

Comment:
this css style class probably deserves a more specific name, since it's not for just any WP content so far.

Reply:
sounds good, renaming this to `lesson`!